java-analyzer-bundle.core/src/main/java/io/konveyor/tackle/core/internal/SymbolInformationTypeRequestor.java

```diff
@@ -62,7 +67,22 @@ public void acceptSearchMatch(SearchMatch match) throws CoreException {
             IJavaElement element = (IJavaElement)match.getElement();
             SymbolKind k = convertSymbolKind(element);
             
+            logInfo("here" + match.getElement().getClass());
```

Comment:
what about this one?

Reply:
I changed this, but yeah that helps when things are not getting symbols, because you know the search is at least finding things